### DISCUSSION TITLE
Filter commits, exclude via regex

### DISCUSSION THREAD

**Discussion by dmaul09** [gitextensions/gitextensions#8174] [General]
I am looking for the correct regex to use in the Filter box in git extensions (v3.3) to exclude all commits containing a certain phrase, i.e., "[Automation]"    To remove the visual clutter of commits I don't care about.  Is there a way?

**Comment by gtoph** [comment]
Looking to do the exact same thing.  Did you ever figure it out?

**Reply by pmiossec** [reply]
It is a hack and it will render very bad in GitExtensions but the syntax is as the following

`PatternToExclude" --invert-grep"`  (the double quote should be written excatly like that!)
